package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "simple-static-website",
+  "version": "1.0.0",
+  "description": "This repository will help beginners who have just completed the Git course &amp; want to apply their theoretical knowledge.",
+  "main": "index.js",
+  "dependencies": {
+    "prettier": "2.0.5"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tushar5526/simple-static-website.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/tushar5526/simple-static-website/issues"
+  },
+  "homepage": "https://github.com/tushar5526/simple-static-website#readme"
+}
```

Comment:
do we need this file?

Reply:
I am not well versed with node.js, but prettier was giving up errors. So I installed it before hand, but I can try a work around in which I install prettier at run time and add files by their extensions, instead of adding all files using `git add .`